docs/src/tutorial/temporal_resolution.md

```diff
@@ -0,0 +1,175 @@
+
+In SpineToolbox, double click on the *InputData*  block.
+
+In the *Object tree* window:
+- Expand "node"
+- Select the electricity node
+- In the *Object parameter value* window, find the value for the electricity node's demand (should be 150 from the Simple System) - right click > Edit\
+	`Parameter type: Time Series fixed resolution`\
+	Copy and paste (or fill in the table):
+	
+		10
+		20
+		40
+		50
+		20
+
+- Click OK - this sets the electricity demand to the variable profile listed above
+
+![image](figs_temporal_resolution/temporal_resolution_time_series_demand.PNG)
+
+### Uniform temporal resolution
+
+#### Setting model start & end timestamps
+
+In the *Object Tree* window:
+- Expand the model
+- Click on "simple"
+
+In the *Object parameter value* window:
+- Add model_start \
+	`parameter_name:` (Double click) `model_start`\
+	`alternative_name:` `Base`\
+	`value:` (Right click) Edit > `Parameter Type: Date Time` > `1 Jan` (default)
+
+- Add model_end\
+	`parameter_name: model_end`\
+	`alternative_name: Base`\
+	`value:` Edit > `Parameter Type: Date Time` > 5 hours after start
+
+Your *Object parameter value* window show look like this:
+
+![image](figs_temporal_resolution/temporal_resolution_model_start_end.PNG)
+
+!!! Regularly save your changes by selecting the *Hamburger menu* (top right) > Commit > Write a message, such as "Save" > OK
+
+#### Setting 1-hr resolution
+
+In the *Object tree* window:
+- Expand temporal_block
+- Select the existing block (should be called "flat" from the Simple System)
+
+In *Object parameter value* window: 
+- Double click `1D` in the value column
+- Change it to `1h`
+- Click OK - this changes the time resolution from daily to hourly
+
+Now we have a model that runs for 5 hours with hourly resolution.
+
+- Save the changes (Commit) and return to the SpineToolbox window
+
+#### Running the model & viewing results
+
+- Click Execute Project (play arrow) and wait for the model to run
+
+- Double click the *OutputData* block
+
+In the *Relationship Tree* window:
+
+- Select report_unit_node_direction_stochastic_scenario
+
+In the *Frozen table* window, select the most recent "Run ..." entry
+
+!!! If the *Frozen table* window is empty, also add the *Pivot table* window using the *Hamburger menu*.
+
+In the *Pivot table* window:
+- Click and drag over all `Time series` in the `unit_flow` column, right click > Plot
+
+You should see different colored lines, for the *electricity* and *fuel* flows that are *to* and *from* the powerplants (red and green are on top of each other). The blue line shows the electricity demand of [10, 20, 40, 50, 20] - and how the fuel demand from Powerplant A is this same array divided by 0.7 (the conversion ratio).
+
+![image](figs_temporal_resolution/temporal_resolution_uniform_results.PNG)
+
+!!! If `Time series` entries are not visible, select the Hamburger menu > View > Pivot Table: Value
+
+### Flexible temporal resolution
+
+#### Creating a time-varying resolution
+
```

Comment:
It is a bit dry if you just say do this do that, perhaps give it a bit of flavour by telling why we would want to do this?

Reply:
That's what I was thinking too.

Reply:
Although I'm not really sure what to say. I added a tiiiiny bit of flavor. Maybe it'll help but let me know if it's still boring.